vaadin-archetype-liferay-portlet/src/main/resources/archetype-resources/src/main/java/__portletClassName__.java

```diff
@@ -0,0 +1,34 @@
+#set( $symbol_pound = '#' )
+#set( $symbol_dollar = '$' )
+#set( $symbol_escape = '\' )
+package ${package};
+
+import javax.portlet.annotations.Dependency;
+import javax.portlet.annotations.PortletConfiguration;
+
+import com.vaadin.flow.component.WebComponentExporter;
+import com.vaadin.flow.portal.VaadinLiferayPortlet;
+import com.vaadin.flow.theme.Theme;
+
+@PortletConfiguration(
+        portletName = "ContactList",
```

Comment:
Should property be used here?
```suggestion
        portletName = "${portletName}",
```

Reply:
Yes, good point